examples/dc/interface/internet.tf

```diff
@@ -0,0 +1,5 @@
+# Get external network with Internet access
```

Comment:
rename the file to base-<smth>

Reply:
base-internet-sprut and base-internet-neutron?

Reply:
this is an example of dc router which is supported on sprut only

Reply:
done